### PR TITLE
Linter: Add default severity-map values

### PR DESCRIPTION
**Pull request by Makman2**

### CODE REVIEW

File: coalib/bearlib/naming_conventions/__init__.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    '__initFile__'
+    >>> to_camelcase('')
+    ''
+    >>> to_camelcase('alreadyCamelCase')
+    'alreadyCamelCase'
+
+    :param string: The string to convert.
+    :return:       The camel-cased string.
+    """
+    # Use [:1] so we can avoid an `if` that checks for a zero-length string.
```
Comment (sils1297) [inline coalib/bearlib/naming_conventions/__init__.py]: outdated comment

File: coalib/bearlib/naming_conventions/__init__.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    '__InitFile__'
+    >>> to_pascalcase('')
+    ''
+    >>> to_pascalcase('AlreadyPascalCase')
+    'AlreadyPascalCase'
+
+    :param string: The string to convert.
+    :return:       The pascal-cased string.
+    """
+    # Use [:1] so we can avoid an `if` that checks for a zero-length string.
```
Comment (sils1297) [inline coalib/bearlib/naming_conventions/__init__.py]: same, otherwise good

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala/pull/2191/commits/05c9d744996068de6487911f6ffa5cdf53b9f2f6

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2191/commits/e555969177ea08312ec7f36f101a47b3c46333d1

**Comment by sils1297** [comment]
@Makman2 rebase next tme please

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2191/commits/b0549b277fb3505ea3771de02a879aa623829f62

**Comment by sils1297** [comment]
rebase and merge please

**Comment by Makman2** [comment]
reack
5f1a600 Introduce bearlib.naming_conventions module
acfd917 Linter: Add default severity-map values

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**